OracleCoherence/README.md

```diff
@@ -1,115 +1,193 @@
+repository. Customers and users are welcome to use them as starters, and customize/tweak, or create
+from scratch new scripts and Dockerfiles.
 
 ## Oracle Linux Base Image
 For more information and documentation, read the [Docker Images from Oracle Linux](https://registry.hub.docker.com/_/oraclelinux/) page.
 
 ### Standalone Distribution
-For more information on the Coherence Standalone Distribution, visit [Coherence 12.2.1 Documentation](http://docs.oracle.com/middleware/12214/coherence/index.html).
+For more information on the Coherence Standalone Distribution, visit the
+[Coherence Documentation](https://docs.oracle.com/en/middleware/standalone/coherence/index.html).
 
 ## Building Oracle JDK (Server JRE) base image
-Before you can build these Oracle Coherence images you must have built the required Oracle Java 8 base image (see [Oracle Java images](../OracleJava/))
+Before you can build these Oracle Coherence images you must have built the required Oracle Java 8
+base image (see [Oracle Java images](../OracleJava/))
 
 ## How to Build
 
-For Coherence 12.2.1.4.0, a Maven project is provided to facilitate building the Docker image.  Maven is used to pull dependent libraries
-which are then bundled into the Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints within the container.
+For Coherence 14.1.1.0.0 and 12.2.1.4.0, a Maven project is provided to facilitate building
+the Docker image.  Maven is used to pull dependent libraries which are then bundled into the
+Docker image to enable running Coherence Management over REST and Coherence Metrics endpoints 
+within the container.
 
-### To build a Coherence 12.2.1.4.0 Docker image
+### To build a Coherence 14.1.1.0.0 or 12.2.1.4.0 Docker image
+
+#### Prerequisites for building with Maven
 
-Prerequisites for building with Maven:
 * Java 8 JDK
 * Maven 3.6.1
 
-Build steps:
+#### Build steps
+
+The following steps build a Coherence 14.1.1.0.0 Docker container as an example
 
 1. Checkout the GitHub Oracle Docker Images repository
-   
-   	`$ git clone git@github.com:oracle/docker-images.git`
+
+    ```shell script
+    git clone git@github.com:oracle/docker-images.git
+    ````
    	
-2. Go to the directory containing the Dockerfile for 12.2.1.4
+1. Go to the directory containing the Dockerfile
 
-    `$ cd OracleCoherence/dockerfiles/12.2.1.4.0/src/main/docker`
+    ```shell script
+    cd OracleCoherence/dockerfiles/14.1.1.0.0/src/main/docker
+    ```
 
-3. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop the Coherence distribution file of your choice into this folder. The build script supports either building an image from either the Standalone Installer, **fmw_12.2.1.4.0_coherence_Disk1_1of1.zip** or the Quick Installer **fmw_12.2.1.4.0_coherence_quick_Disk1_1of1.zip**
+1. [Download](https://www.oracle.com/middleware/technologies/coherence-downloads.html) and drop
```

Comment:
I would recommend rewriting this line as "`[Download the Coherence distribution file](https://www.oracle.com/middleware/technologies/coherence-downloads.html) of your choice and save it to the current directory`" instead so that the link is more accessible for screen readers.